webpack.config.js

```diff
@@ -61,17 +61,34 @@ module.exports = {
                     {
                         loader: "babel-loader",
                         options: {
+                            sourceType: "unambiguous",

```

Comment:
Needed? Where was it copied from?

Reply:
uxf_webpack, which is being used by ripe-components-vue